[2020-06-03 18:10:53,190] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: hello_world.second_task 2020-06-03T12:38:03.824519+00:00 [queued]>
[2020-06-03 18:10:53,199] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: hello_world.second_task 2020-06-03T12:38:03.824519+00:00 [queued]>
[2020-06-03 18:10:53,199] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 18:10:53,199] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-03 18:10:53,199] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 18:10:53,208] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): second_task> on 2020-06-03T12:38:03.824519+00:00
[2020-06-03 18:10:53,213] {standard_task_runner.py:53} INFO - Started process 17653 to run task
[2020-06-03 18:10:53,256] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: hello_world.second_task 2020-06-03T12:38:03.824519+00:00 [running]> shreya-nineleaps
[2020-06-03 18:10:53,271] {logging_mixin.py:112} INFO - Unnamed: 0     int64
states        object
cases          int64
date          object
dtype: object
[2020-06-03 18:10:53,273] {taskinstance.py:1145} ERROR - Missing optional dependency 'pandas_gbq'. pandas-gbq is required to load data from Google BigQuery. See the docs: https://pandas-gbq.readthedocs.io. Use pip or conda to install pandas_gbq.
Traceback (most recent call last):
  File "/home/nineleaps/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 983, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/nineleaps/anaconda3/lib/python3.7/site-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/home/nineleaps/anaconda3/lib/python3.7/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/nineleaps/airflow/dags/hello_world.py", line 41, in second_function
    table_schema=None)
  File "/home/nineleaps/anaconda3/lib/python3.7/site-packages/pandas/core/frame.py", line 1523, in to_gbq
    private_key=private_key,
  File "/home/nineleaps/anaconda3/lib/python3.7/site-packages/pandas/io/gbq.py", line 195, in to_gbq
    pandas_gbq = _try_import()
  File "/home/nineleaps/anaconda3/lib/python3.7/site-packages/pandas/io/gbq.py", line 12, in _try_import
    pandas_gbq = import_optional_dependency("pandas_gbq", extra=msg)
  File "/home/nineleaps/anaconda3/lib/python3.7/site-packages/pandas/compat/_optional.py", line 93, in import_optional_dependency
    raise ImportError(message.format(name=name, extra=extra)) from None
ImportError: Missing optional dependency 'pandas_gbq'. pandas-gbq is required to load data from Google BigQuery. See the docs: https://pandas-gbq.readthedocs.io. Use pip or conda to install pandas_gbq.
[2020-06-03 18:10:53,275] {taskinstance.py:1202} INFO - Marking task as FAILED.dag_id=hello_world, task_id=second_task, execution_date=20200603T123803, start_date=20200603T124053, end_date=20200603T124053
[2020-06-03 18:11:03,185] {logging_mixin.py:112} INFO - [2020-06-03 18:11:03,184] {local_task_job.py:103} INFO - Task exited with return code 1
